gce-containers-startup/gce-containers-startup_test.go

```diff
@@ -15,19 +15,21 @@
 package main
 
 import (
-	"testing"
-	utils "github.com/konlet/utils"
 	"io"
 	"io/ioutil"
 	"strings"
+	"testing"
+
+	utils "github.com/konlet/utils"
+
+	"fmt"
```

Comment:
Why isn't this sorted in the list above if you're already making such changes?

Reply:
Unsure, my vim is doing some automated crazyness here. Anyway, fixing.